modules/_labs/header/react/lib/Header.tsx

```diff
@@ -104,7 +105,7 @@ const navStyle = ({themeColor}: Pick<HeaderProps, 'themeColor'>) => {
   return css({
     nav: {
       display: 'flex',
-      flexGrow: 1,
+      flex: `1 0 auto`, // Instead of just flex-grow: 1 for IE11, see https://github.com/philipwalton/flexbugs#flexbug-1
       justifyContent: 'center',
       height: 'inherit',
       marginLeft: spacing.xl,
```

Comment:
I think this margin left shouldn't be here when the menu is suppose to be centered, otherwise it is pushed a bit right

Reply:
I can't find the specs anymore but at one time it was probably used to compensate for all the margins of the branding. I'll explicitly set it to 0 here (otherwise it sets to 16px somewhere else)

Reply:
Oh actually it's for spacing against the `leftSlot`. I'll make the `ChildrenSlot` `leftSlot` aware so if it's been included, it'll have a 40px margin-left... if not it'll be flush (0px margin-left).

Reply:
nvm folks, this margin is for many things, we'll have to think of the header more holistically, I'll open an issue for this that we can track later.